src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/rules-engine.mdx

```diff
@@ -121,7 +121,7 @@ The `Maximum Burst size` is only available for the `Req/s` (request per second).
 :::
 
 :::caution[Warning]
-Azion platform maintains only one `Set WAF Rule Set` behavior configuration for each criteria logic. If you have two different Edge Firewall Rule Sets configured with the same criteria logic but different `Set WAF Rule Set` behaviors, only the most recent one will be processed. This can be useful in case your application needs constant switching between two or more behaviors for the same criteria.
+Azion platform maintains only one `Set WAF Rule Set` behavior configuration for each criteria logic. If you have two different edge firewalls configured with the same criteria logic but different `Set WAF Rule Set` behaviors, only the most recent one will be processed. This can be useful in case your application needs constant switching between two or more behaviors for the same criteria.
```

Comment:
```suggestion
Azion's platform maintains only one `Set WAF Rule Set` behavior configuration for each criteria logic. If you have two different edge firewalls configured with the same criteria logic but different `Set WAF Rule Set` behaviors, only the most recent one will be processed. This can be useful if your application constantly switches between two or more behaviors for the same criteria.
```